Openshift/templates/entando-eap71-jenkins-quickstart.yml

```diff
@@ -5,17 +5,17 @@ metadata:
   annotations:
     iconClass: icon-eap
     tags: eap,javaee,java,jboss
-    version: 5.0.1
+    version: 5.0.1-SNAPSHOT
```

Comment:
Make sense to externalize this?

Reply:
Good idea: https://jira.entando.org/browse/EN-1887